pages/common/gifdiff.md

```diff
@@ -0,0 +1,13 @@
+# gifdiff
+
+> `diff` for GIFs.
```

Comment:
Welcome to tldr @CairnThePerson,  I think we can reword this line a bit as mentioned [here](https://github.com/kohler/gifsicle).
```suggestion
> Compare two GIFs for identical visual appearance. 
```

Reply:
That's fair!